src/features/network/store.ts

```diff
@@ -1,7 +1,7 @@
-import create from "zustand";
-import { persist } from "zustand/middleware";
-import localforage from "localforage";
```

Comment:
Alberto's style guide (see `.prettierrc`) disables semi-colons but somehow this file had them.